lib/themes/dosomething/paraneue_dosomething/scss/content/_reportback-permalink.scss

```diff
@@ -0,0 +1,58 @@
+.reportback__permalink {
+  .header {
+    > .wrapper {
+      text-align: left;
+    }
+  }
+
+  .container {
+    &.-purple {
+      background: $purple;
+    }
+
+    .-white {
+      background: $white;
+    }
+
+    > .wrapper {
+      @include span(16);
```

Comment:
The mobile design has spacing around the white background, the default display spanned the white container 100%. 

![image](https://cloud.githubusercontent.com/assets/1700409/6508201/6230860e-c325-11e4-880e-a19ac82511fe.png)